sdk/ai/azure-ai-inference/README.md

```diff
@@ -534,6 +542,88 @@ To report issues with the client library, or request additional features, please
 
 * Have a look at the [Samples](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/ai/azure-ai-inference/samples) folder, containing fully runnable Python code for doing inference using synchronous and asynchronous clients.
 
+## Tracing
```

Comment:
I think this order of sections makes more sense:

## Troubleshooting
## Tracing
## Next steps
## Contributing

Reply:
okay.  i had it like that at one point, but then changed it before committing the change